flask_talisman/talisman.py

```diff
@@ -136,6 +144,8 @@ def init_app(
                 'Setting content_security_policy_report_only to True also '
                 'requires a URI to be specified in '
                 'content_security_policy_report_uri')
+        self.content_security_policy_nonce_in = \
```

Comment:
don't use `\` for line continuations, use `()`.

Reply:
Good point - this would make the updates inconsistent with the rest of the code. Does that matter? Should I update the other uses of `\`?

Reply:
Eh, nah. We can address those later I just don't want this to proliferate.

Reply:
Yep fair enough - done!